js/main.js

```diff
@@ -1,11 +1,17 @@
-import {createPhotos} from './mockPhotos.js';
-import {renderPhotos} from './renderingPicture.js';
-import { setPhotos} from './popupGallery.js';
-import './form.js';
-import './photoScale.js';
-import './photoFilters.js';
-
-const photos = createPhotos();
+import {load,setSubmit} from './api.js';
+import {renderPhotos} from './rendering-picture.js';
+import { setPhotos} from './popup-gallery.js';
+import {setUserFormSubmit} from './form.js';
+import { showAlert } from './alert.js';
+import './photo-scale.js';
+import './photo-filters.js';
+
+const photos = await load();
+
 renderPhotos(photos);
```

Comment:
Добавим проверку, ведь фотографий может и не быть, если они не загрузятся
```suggestion
if (photos) {
  renderPhotos(photos);
  setPhotos(photos);
}
```